packages/shylu/shylu_node/basker/src/shylubasker_order_match.hpp

```diff
@@ -2,31 +2,31 @@
 #define SHYLUBASKER_ORDER_MATCH_HPP
 
 
-//#define BASKER_MC64
-
 //Include statements
-#ifdef BASKER_MC64
-#include "mc64ad.hpp"
-#endif
-
-#include "mwm2.hpp"
-
-
+//#include "mc64ad.hpp"
 /* Note, come back and rewrite these 
  * function yourself at somepoint to clean up
  * In the mean time to test how well this works, we will use the auto rewritten from SuperLU_dist with modifications
  */
 
-/*
+#include "mwm2.hpp"
+//#if defined(HAVE_AMESOS2_SUPERLUDIST) && !defined(BASKER_MC64)
+//  #define BASKER_SUPERLUDIS_MC64
```

Comment:
Where can this macro be defined? I see guarded code paths based on this, but unclear if we have option to set (I didn't see it in the config definitions)